### ISSUE TITLE
Change mouse hover 

### ISSUE THREAD

**Issue by Tamir198** [CyBear-Jinni/CBJ_Site#156]
Can we have a hover icon when hovering the tabs with the mouse?

![image](https://github.com/CyBear-Jinni/CBJ_Site/assets/34707669/ec87aca1-9d30-4f37-8fb5-e4791de2ff50)

**Comment by guyluz11** [comment]
Sure, I think you should expand the explanation.
You are talking about the mouse hover icon, not the label icon right?

**Comment by Tamir198** [comment]
Yes, the mouse hover icon should display when hovering on one of the tabs
